components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/oauth-login.jsp

```diff
@@ -178,7 +178,7 @@
                                                 }
                                             %>
                                             <input type="button" class="button"
-                                                   onclick="javascript:location.href='<%=forwardUrl%>'"
+                                                   onclick="javascript:location.href='<%=Encode.forHtmlAttribute(forwardUrl)%>'"
```

Comment:
For href " We should validate at the input whether it conforms to URI pattern" as we have done in [1]
[1] https://github.com/wso2-support/identity-apps/pull/212